models/basemodel.py

```diff
@@ -275,6 +275,18 @@ def unsubscribe_user(self, user_id, row_id, user_subscription_name=None):
                 logging.warn("row_id {0} not in user {1}".format(row_id, user_subscription_name))
                 pass
             r.db(self.DB).table(user_table).get(user_id).update({user_subscription_name: user_subscription}).run(self.conn)
+        logging.info("ROW_ID IS {0}".format(row_id))
+        active_surveys = r.db(self.DB).table('Group').get(row_id).getField('active_surveys').run(self.conn)
```

Comment:
https://rethinkdb.com/api/python/get_field/

The python API is different from the Javascript API.